packages/analytics/dashboard-renderer/vite.config.ts

```diff
@@ -18,11 +18,12 @@ const config = mergeConfig(sharedViteConfig, defineConfig({
     },
     rollupOptions: {
       // Make sure to externalize deps that shouldn't be bundled into your library
-      external: ['@kong-ui-public/analytics-chart'],
+      external: ['@kong-ui-public/analytics-chart', 'swrv'],
```

Comment:
I externalized `swrv` because it seemed silly to bump the the bundle size limit.